src/conf.R

```diff
@@ -287,6 +287,7 @@ AddDescription = function(short, long) {
 
 AddNodeType("BGK","COLLISION")
 AddNodeType("MRT","COLLISION")
+AddNodeType("CM","COLLISION")
```

Comment:
W would not want this by default. In fact as we go I would not want any NodeTypes by default, and all defined in models.